roles/static_files/templates/etc/nginx/sites-available/static_files

```diff
@@ -8,12 +8,15 @@ server {
     listen   8080;
     listen   [::]:8080 default ipv6only=on;  ## listen for ipv6
     port_in_redirect off;
+    expires          -1;
 
     location /specials {
+        expires    1m;
         add_header Access-Control-Allow-Origin $cors_header;
     }
 
     location ~ ^/files/(.+)$ {
+        expires   1y;
```

Comment:
this would be great, except for the dang PDFs that get replaced with the same filename when a new one is uploaded. Again, expires controls recheck, so we could live w/ shorter, as long as we implement ETAGs  as well.